agent/command.go

```diff
@@ -66,6 +67,15 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 			grip.Error("runCommands canceled")
 			return errors.New("runCommands canceled")
 		}
+		if cmd.DisplayName() == "" {
+			grip.Critical(message.Fields{
```

Comment:
Should this be logged by the logger instead? It would make it easier to get to.

Reply:
My intent was not to make this visible to users since there's nothing they can do and I want to see some internal data structures that wouldn't mean anything to users. If we think it's useful to inform them that something went wrong, I can log to both